docs/spec/vesting/TODO.md

```diff
@@ -0,0 +1 @@
+I believe this spec already exists somewhere
```

Comment:
Maybe we find where it is and move it to the appropriate place? May need to rebase this PR on `develop` to pull in the vesting spec.

Reply:
It's in the `auth` module spec, as that's where the code lives.

Reply:
k deleting this folder